src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingService.kt

```diff
@@ -589,6 +596,18 @@ class SentencingService(
     }
   }
 
+  private fun findConsecutiveSentenceLineSequence(sentenceSequence: Long, offenderBooking: OffenderBooking): Long {
+    val sentence = offenderSentenceRepository.findByIdOrNull(
+      SentenceId(
+        sequence = sentenceSequence,
+        offenderBooking = offenderBooking,
+      ),
+    )
+      ?: throw NotFoundException("Consecutive sentence for booking ${offenderBooking.bookingId} and sentence sequence $sentenceSequence not found")
+    return sentence.lineSequence?.toLong()
+      ?: throw NotFoundException("No Line sequence found for Consecutive sentence with booking ${offenderBooking.bookingId} and sentence sequence $sentenceSequence")
```

Comment:
no line sequence should a "can't happen" for this type of sentence - just making sure the exception is explicit if it does